src/components/FooterPreLogin/FooterPreLogin.tsx

```diff
@@ -23,7 +23,7 @@ import { ProductArrayType } from "./ProductType";
 type FooterPreLoginProps = LangSwitchProps & {
   companyLink: CompanyLinkType;
   links: PreLoginFooterLinksType;
-  onExit?: (href: string, linkType: LinkType) => void;
+  onExit?: (linkType: LinkType, href?: string) => void;
```

Comment:
it's not recommended to change the signature of a method inside a Prop, you risk to break all the application that are using it!